documentation/guide-jpa.asciidoc

```diff
@@ -393,18 +394,41 @@ The environment specific settings (especially passwords) are configured by the o
 
 For further details about `database.hibernate.hbm2ddl.auto` please see http://docs.jboss.org/hibernate/orm/5.0/manual/en-US/html/ch03.html#configuration-misc-properties[here]. For production and acceptance environments we use the value `validate` that should be set as default. In case you want to use Oracle RDBMS you can find additional hints link:guide-oracle.asciidoc#driver[here].
 
+.**quarkus**
+----
+  quarkus.datasource.jdbc.url=jdbc:postgresql://database.enterprise.com/app
+  quarkus.datasource.username=appuser01
+  quarkus.datasource.password=************
+  quarkus.datasource.db-kind=postgresql
+
+  # drop and create the database at startup (use only for local development)
+  quarkus.hibernate-orm.database.generation=drop-and-create
+----
+
+For Quarkus applications you additionally have to add the respective JDBC driver extension. There are different drivers for different database types. See https://quarkus.io/guides/hibernate-orm#setting-up-and-configuring-hibernate-orm[Quarkus Hibernate guide].
+
```

Comment:
Once again: I think this subsection really differs according to the framework. This generally applies for the entire configuration guide.
So maybe we should remove all the content here and just say "For spring read here and for quarkus read here" both deep-linking to the according config guide specific for the framework.

Reply:
I moved this section to the configuration guide and added a new configuration guide for Quarkus ([Spring](https://github.com/GuentherJulian/devon4j/blob/jpa_guide/documentation/guide-configuration.asciidoc#database-configuration), [Quarkus](https://github.com/GuentherJulian/devon4j/blob/jpa_guide/documentation/quarkus/guide-configuration.asciidoc))
Doesn't the 'Database Configuration' subsection belong in the 'Externalized Configuration' section rather than the 'Internal Application Configuration' section?